composer.json

```diff
@@ -22,6 +22,7 @@
         }
     },
     "scripts": {
+        "cghooks": "vendor/bin/cghooks",
```

Comment:
This should [allow running](https://github.com/brainmaestro/composer-git-hooks#shortcut) `composer cghooks`, instead of relying on `cghooks` to be installed in the system path. It should address the Travis error.